contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_checkstyle.py

```diff
@@ -5,6 +5,7 @@
 import subprocess
 import sys
 from contextlib import contextmanager
+from pathlib import Path
```

Comment:
This file is only a refactor to make it easier to eventually get this working with `--chroot`.